content/references/http-websocket-apis/public-api-methods/server-info-methods/nft_info.md

```diff
@@ -0,0 +1,168 @@
+---
+html: nft_info.html
+parent: server-info-methods.html
+blurb: Retrieve status of the specified NFT in human-readable format.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# nft_info
+[[Source]](https://github.com/XRPLF/clio/blob/4a5cb962b6971872d150777881801ce27ae9ed1a/src/rpc/handlers/NFTInfo.cpp "Source")
+
+The `nft_info` command asks the Clio server for a human-readable version of information about [the NFT](non-fungible-tokens.html) being queried. 
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 1,
+  "command": "nft_info",
+  "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+  "ledger_index": "current"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "nft_info",
+    "params": [
+      {
+          "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+          "ledger_index": "current"
+      }
+    ]
+}
+```
+
+*Commandline*
+
+```sh
+#Syntax: server_info
+clio nft_info 0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C current
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+[Try it! >](websocket-api-tool.html#nft_info)
+
+The request contains the following parameters:
+
+| `Field`        | Type                       | Description                    |
+|:---------------|:---------------------------|:-------------------------------|
+| `nft_id`       | String                     | A unique identifier for the non-fungible token (NFT). |
+| `ledger_hash`  | String                     | _(Optional)_ A 20-byte hex string for the ledger version to use. (See [Specifying Ledgers][]) |
+| `ledger_index` | String or Unsigned Integer | _(Optional)_ The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) | 
```

Comment:
we need to call out that users cannot use `ledger_index: current` for this request. doing so will forward the request to `rippled`, but `rippled` does not know this API.

Reply:
also cannot use `closed` as a value here. can only use `validated` or a ledger number

Reply:
we could also say that not specifying `ledger_hash` or `ledger_index` is equivalent to passing `validated`